actors/miner/tests/miner_actor_test_commitment.rs

```diff
@@ -608,12 +630,9 @@ mod miner_actor_test_commitment {
         let expiration =
             deadline.period_end() + DEFAULT_SECTOR_EXPIRATION * rt.policy.wpost_proving_period;
         {
-            // After version 7, only V1_1 accepted
-            rt.set_network_version(NetworkVersion::V8);
-
             let mut precommit_params =
                 h.make_pre_commit_params(104, challenge_epoch, expiration, vec![]);
-            precommit_params.seal_proof = RegisteredSealProof::StackedDRG32GiBV1;
+            precommit_params.seal_proof = RegisteredSealProof::StackedDRG2KiBV1;
```

Comment:
Keep 32GiBV1 don't switch to 2k which will could be failing for other reasons (mismatched wpost proof type)

Reply:
I'm not able to make the call to `pre_commit_sector` fail if I'm not using the `StackedDRG2KiBV1` proof. Is `StackedDRG32GiBV1` supposed to make this call failing?

Reply:
Yes it is

Reply:
All 2k proofs are disallowed by default but this test is checking that V1 proofs of the allowed size are also disallowed.  32GiBV1_1 is the only thing that should work

Reply:
I also see that this does not error.  Looking into it now. I suspect the testing harness...

Reply:
Ok thanks! Let me know, I can investigate on it as well.

Reply:
@elmattic I figured it out.  The new_runtime function erroneously sets 32GiBV1 as the default proof type.  If you change that to 32GiBV1P1 this and all other miner tests pass locally for me.